cmd/tailscale/cli/exitnode.go

```diff
@@ -97,8 +108,34 @@ func runExitNodeList(ctx context.Context, args []string) error {
 	}
 	fmt.Fprintln(w)
 	fmt.Fprintln(w)
-	fmt.Fprintln(w, "# To use an exit node, use `tailscale set --exit-node=` followed by the hostname or IP")
+	fmt.Fprintln(w, "# To use an exit node, use `tailscale set --exit-node=` followed by the hostname or IP. Or `tailscale exit-node suggest` to have Tailscale pick an exit node for you.")
```

Comment:
thoughts on `tailsacle set --exit-node=auto` instead?

To me, "suggest" is a command to offer a recommendation, not a Google "I'm feeling lucky" that just picks and does it without confirmation.

Reply:
Agree with what Adrian said here.